src/main/java/de/tum/in/www1/artemis/web/rest/SubmissionResource.java

```diff
@@ -65,6 +65,7 @@ public SubmissionResource(SubmissionRepository submissionRepository, ResultServi
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
     @PostMapping("/submissions")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
```

Comment:
I didn't find any usage of the endpoints except the delete one. They didn't have any authorization checks and in my opinion can cause security issues. I would suggest to remove them. Should I do that @krusche ?

Reply:
yes, please remove the unused endpoints, but double check that they are really unused